radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/AtomObservation.java

```diff
@@ -1,9 +1,22 @@
 package com.radixdlt.client.core.atoms;
 
+import com.radixdlt.client.core.atoms.AtomEvent.AtomEventType;
+
 public class AtomObservation {
```

Comment:
Why do we need AtomObservation? Is it that different from what the AtomEvent variants do?

Reply:
It has the HEAD state in it as well.